scribble-lib/scribble/acmart.rkt

```diff
@@ -367,11 +371,29 @@
 (define (email . text)
   (author-email text))
 
+(define (email-string . text)
+  (define text-escaped
+    (for/list ([str (in-list text)])
+      (escape-email-string str)))
+  (author-email
+   (list
+    (make-element
+     (make-style #f '(exact-chars))
+     text-escaped))))
+
 (define (convert-email email)
   (make-element
    (make-style "SAuthorEmail" command-props)
    (decode-content (email-text email))))
 
+(define escape-email-map
+  #(("#" . "\\#")
+    ("%" . "\\%")))
+(define (escape-email-string str)
+  (for/fold ([str str])
+            ([escape-map (in-vector escape-email-map)])
```

Comment:
Why not use a hash and `in-hash`?

(The PR looks good tho)

Reply:
There's not a reason for the choice of a vector. (Maybe because the code doesn't need to lookup which character is mapped to what.)